src/components/ReturnButton.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="back-button" @click="$router.go(-1)">
```

Comment:
You can use just back method
```suggestion
  <div class="back-button" @click="$router.back">
```

Reply:
And try to do semantic markup. Eg Button tags for buttons.
Since we use vuetify you can use button component from vuetify library